### PR TITLE
add InSpec CLI reference topic

### PR DESCRIPTION
**Pull request by jamescott**

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: docs/ctl_inspec.rst [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+will return:
+
+.. code-block:: javascript
+
+   {"family":"ubuntu","release":"14.04","arch":null}
+
+
+exec
+=====================================================
+Use ``inspec json`` to read all tests at the specified path, and then generate a |json| profile to standard output (stdout) or printed to the command shell.
```
Comment (chris-rock) [inline docs/ctl_inspec.rst:95]: @jamescott this should be `inspec exec`, right?

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: docs/ctl_inspec.rst (hunk 1 above)
Comment (chris-rock) [inline docs/ctl_inspec.rst:95]: As discussed with @jamescott we fix this in a new PR

**Comment by chris-rock** [comment]
That is a great addition @jamescott , now: @delivery approve

**Comment by chef-delivery** [comment]
Change: e9dd2e38-3d30-4e09-adc4-d6d0026709e3 approved by: @chris-rock

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**